pegasus/sites.v3/code.org/views/video_cast.haml

```diff
@@ -0,0 +1,8 @@
+- people.each do |person|
+  .row.film-cast-member
+    %h2.col-xs-12{style: "margin-top: 1em; margin-bottom: .75em; padding-bottom: .5em; border-bottom: 1px solid #b7b7a7;"} #{person[:name_s]}
```

Comment:
At this point, I'm not certain; we're still waiting on content for both pages, and I would not be surprised to see the two tables take different forms.